synapse/config/emailconfig.py

```diff
@@ -98,21 +97,18 @@ def read_config(self, config, **kwargs):
             if parsed[1] == "":
                 raise RuntimeError("Invalid notif_from address")
 
-        template_dir = email_config.get("template_dir")
-        # we need an absolute path, because we change directory after starting (and
+        # A user-configurable template directory. Templates will first be looked for here,
+        # and if not found, will be taken from the default template directory instead.
+        template_dir = email_config.get("template_dir") or self.default_template_dir
```

Comment:
Why do we need the `self.default_template_dir` here? It seems if `template_dir` is false-y we should just let `email_tempalte_dir` be false-y?

Reply:
Looks like this was left over from the first attempt to use the default template dir instead. Thanks for catching, I've removed it :slightly_smiling_face: